runner/src/scenario/ceramic/model_instance.rs

```diff
@@ -688,7 +688,6 @@ impl ModelInstanceRequests {
         let name = format!("create_index_model_{}", tx_name);
         let req = cli
             .create_index_model_request(model_id, &resp.code)
-            .await
```

Comment:
Why did we remove the await here? Isn't this method async?

Reply:
not anymore. the signer used by ceramic_core changed in [this PR](https://github.com/ceramicnetwork/rust-ceramic/pull/357/files#diff-ea616cf3110776cfec65245b2c2def84dd0f227005583ca991b8755838fa54c3) and isn't async anymore.. so the http client was updated and finally this one

Reply:
you should get a warning if you don't ever await a future, which should cause our CI to fail since we deny warnings